themes/digital.gov/layouts/guides/list.html

```diff
@@ -7,31 +7,25 @@
 
 
   <main role="main" id="main-content">
-    <article>
-      <div class="grid-container grid-container-desktop">
-        {{- partial "partials/core/usa-breadcrumbs.html" . -}}
-        <div class="grid-row grid-gap-4">
-          <div class="grid-col-12 tablet-lg:grid-col-2">
-            {{- partial "core/guides/guide-nav.html" . -}}
-          </div>
+    <article class="grid-container grid-container-desktop">
+      {{- partial "partials/core/usa-breadcrumbs.html" . -}}
+      <section class="grid-row grid-gap-4">
+        {{- partial "core/guides/guide-nav.html" . -}}
+
 
-          <div
-            class="grid-col-12 tablet-lg:grid-col-8"
-            data-edit-this="{{- $path -}}"
-          >
-            <div class="usa-layout-docs__main usa-prose">
-              {{- partial "core/guides/guide-page-head.html" . -}}
+        <div class="grid-col-12 tablet-lg:grid-col-8">
```

Comment:
## Optional

Thanks to the `tablet-lg` prefix we can leave out `grid-col-12`. It'll default to full width **until** it reaches `tablet-lg`.

```suggestion
        <div class="tablet-lg:grid-col-8">
```